tests/system/action/base.py

```diff
@@ -364,7 +364,7 @@ def assert_history_information(
         """
         informations = self.datastore.history_information([fqid])[fqid]
         if information is None:
-            assert information is None
+            assert not informations
```

Comment:
why does `is None` not work?

Reply:
is None does not work?

Reply:
The delete_history_information results in an empty list and not in None.
We didn't catch this in the delete history information, because there the test was broken.
So I fixed this check.